cmd/protoc-gen-go-grpc/grpc.go

```diff
@@ -187,6 +187,13 @@ func generateFileContent(gen *protogen.Plugin, file *protogen.File, g *protogen.
 	}
 }
 
+func copyCommentsFromProto(g *protogen.GeneratedFile, service *protogen.Service) {
+	// add line for vscode dosn't display top of comment
```

Comment:
Let's not mention vscode in our comments, please.  We're not coding for vscode; we're coding for the godoc output.  If vscode doesn't work well with standard godoc then it needs to change, not us.